sphinxcontrib_autodoc_filterparams/__init__.py

```diff
@@ -34,14 +34,14 @@ def process_docstring(app, what, name, obj, options, lines):
 
 def process_signature(app, what, name, obj, options,
                       signature, return_annotation):
-    filter_params = getattr(app.config, _ext, None)
+    filter_params = ((getattr(app.config, _ext, None)()),)
```

Comment:
I'm getting below error: Handler <function process_signature at 0x000001AADCA3E700> for event 'autodoc-process-signature' threw an exception (exception: 'type' object is not iterable)
in case, it's not called as single-element tuple

then filter_params  fun will request 2 positional params
def sphinxcontrib_autodoc_filterparams(fun='', param=''):
    return not param.startswith('_')